test/sample-test/sample_test_launcher.py

```diff
@@ -0,0 +1,262 @@
+        target_image_prefix=target_image_prefix,
+        namespace=namespace
+    )
+    self._dataflow_tft_image = dataflow_tft_image
+    self._dataflow_predict_image = dataflow_predict_image
+    self._dataflow_tfma_image = dataflow_tfma_image
+    self._dataflow_tfdv_image = dataflow_tfdv_image
+    self._dataproc_create_cluster_image = dataproc_create_cluster_image
+    self._dataproc_delete_cluster_image = dataproc_delete_cluster_image
+    self._dataproc_analyze_image = dataproc_analyze_image
+    self._dataproc_transform_image = dataproc_transform_image
+    self._dataproc_train_image = dataproc_train_image
+    self._dataproc_predict_image = dataproc_predict_image
+    self._kubeflow_dnntrainer_image = kubeflow_dnntrainer_image
+    self._kubeflow_deployer_image = kubeflow_deployer_image
+    self._local_confusionmatrix_image = local_confusionmatrix_image
+    self._local_roc_image = local_roc_image
+
+  def _injection(self):
+    """Sample-specific image injection into yaml file."""
+    subs = {
+        'gcr\.io/ml-pipeline/ml-pipeline/ml-pipeline-local-confusion-matrix:\w+':self._local_confusionmatrix_image,
+        'gcr\.io/ml-pipeline/ml-pipeline/ml-pipeline-local-roc:\w+':self._local_roc_image
+    }
+    if self._test_name == 'xgboost_training_cm':
+      subs.update({
+          'gcr\.io/ml-pipeline/ml-pipeline-dataproc-create-cluster:\w+':self._dataproc_create_cluster_image,
+          'gcr\.io/ml-pipeline/ml-pipeline-dataproc-delete-cluster:\w+':self._dataproc_delete_cluster_image,
+          'gcr\.io/ml-pipeline/ml-pipeline-dataproc-analyze:\w+':self._dataproc_analyze_image,
+          'gcr\.io/ml-pipeline/ml-pipeline-dataproc-transform:\w+':self._dataproc_transform_image,
+          'gcr\.io/ml-pipeline/ml-pipeline-dataproc-train:\w+':self._dataproc_train_image,
+          'gcr\.io/ml-pipeline/ml-pipeline-dataproc-predict:\w+':self._dataproc_predict_image,
+      })
+
+      utils.file_injection('%s.yaml' % self._test_name,
+                           '%s.yaml.tmp' % self._test_name,
+                           subs)
+    elif self._test_name == 'tfx_cab_classification':
+      subs.update({
+          'gcr\.io/ml-pipeline/ml-pipeline-dataflow-tft:\w+':self._dataflow_tft_image,
+          'gcr\.io/ml-pipeline/ml-pipeline-dataflow-tf-predict:\w+':self._dataflow_predict_image,
+          'gcr\.io/ml-pipeline/ml-pipeline/ml-pipeline-dataflow-tfdv:\w+':self._dataflow_tfdv_image,
+          'gcr\.io/ml-pipeline/ml-pipeline/ml-pipeline-dataflow-tfma:\w+':self._dataflow_tfma_image,
+          'gcr\.io/ml-pipeline/ml-pipeline/ml-pipeline-kubeflow-tf-trainer:\w+':self._kubeflow_dnntrainer_image,
+          'gcr\.io/ml-pipeline/ml-pipeline/ml-pipeline-kubeflow-deployer:\w+':self._kubeflow_deployer_image,
+          'gcr\.io/ml-pipeline/ml-pipeline/ml-pipeline-local-confusion-matrix:\w+':self._local_confusionmatrix_image,
+          'gcr\.io/ml-pipeline/ml-pipeline/ml-pipeline-local-roc:\w+':self._local_roc_image
```

Comment:
These two substitutions are already included, aren't they?

Reply:
Good catch. Forgot to remove. My bad.